### DISCUSSION TITLE
Does Maxwells Daemon support Oauth2 authentication?

### DISCUSSION THREAD

**Discussion by setourneux** [zendesk/maxwell#2075] [General]
Does Maxwells Daemon support Oauth2 authentication?

For a project our kafka infrastructure will force the use of Oauth2 for all producers. So I wonder if Maxwells supports it.

**Comment by osheroff** [comment]
according to https://docs.cloudera.com/cdp-private-cloud-base/7.1.8/kafka-securing/topics/kafka-securing-oauth-clients.html, everything is just configured through the properties file, so there shouldn't be any reason why Maxwell doesn't support it.  Having said that, I've never set it up personally.
